ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/BlockFetch/Server.hs

```diff
@@ -75,14 +75,14 @@ blockFetchServer
     -> ChainDB m blk
     -> NodeToNodeVersion
     -> ResourceRegistry m
-    -> BlockFetchServer (Serialised blk) m ()
+    -> BlockFetchServer (Serialised blk) (Point (Serialised blk)) m ()
```

Comment:
If you prefer, we can turn this into `Point blk` with some calls to `castPoint` in the right places.

Reply:
I think that'd be reasonable. I wanted to start with the mechanical change, or things rapidly get very confusing.